src/main/java/com/google/api/codegen/transformer/py/PythonSurfaceNamer.java

```diff
@@ -453,4 +453,27 @@ public String getReleaseAnnotation(ReleaseLevel releaseLevel) {
         throw new IllegalStateException("Invalid development status");
     }
   }
+
+  @Override
+  public String getPrintSpec(String spec) {
```

Comment:
We should test this. I'm fine with adding a `TODO(pongad)` for now

Reply:
I'll just add it in the todo table in the doc.